modules/RTC/TraceablePeerConnection.js

```diff
@@ -1070,6 +1097,16 @@ const getters = {
                 'getLocalDescription::postTransform (munge local SDP)', desc);
         }
 
+        // What comes out of this getter will be signalled over Jingle to
+        // the other peer and we need to make sure that the media direction is
+        // 'sendrecv', because no changes are not signaled, after the initial
+        // offer/answer exchange.
+        //
+        // Note the this is not the effective media direction which will be set
+        // on the peerconnection - it's being adjusted to the correct value in
+        // the local description setter.
```

Comment:
How about:

> 
> What comes out of this getter will be signalled over Jingle to the other peer, so we need to make sure the media direction is 'sendrecv' because we won't change the direction later and don't want the other peer to think we can't send or receive.
> 
> Note that the description we set in chrome does have the accurate direction (e.g. 'recvonly'), since that is technically what is happening.